flink-python/pyflink/datastream/functions.py

```diff
@@ -147,3 +150,34 @@ def _get_python_env():
     gateway = get_gateway()
     exec_type = gateway.jvm.org.apache.flink.table.functions.python.PythonEnv.ExecType.PROCESS
     return gateway.jvm.org.apache.flink.table.functions.python.PythonEnv(exec_type)
+
+
+class JavaFunctionWrapper(object):
+
+    def __init__(self, j_function):
+        self._j_function = j_function
+
+    def get_java_function(self):
+        return self._j_function
+
+
+class SinkFunction(JavaFunctionWrapper):
+    """
+    The base class for SinkFunctions.
+    """
+
+    def __init__(self, sink_func: Union[str, JavaObject], *args):
```

Comment:
If you want to support `*args`, you need to take all types into consideration, i.e., convert different python types to java types. I think we don't need to support `*args` here.